plugins/woocommerce/tests/e2e-pw/tests/admin-marketing/overview.spec.js

```diff
@@ -4,24 +4,34 @@ const { tags } = require( '../../fixtures/fixtures' );
 test.describe( 'Marketing page', () => {
 	test.use( { storageState: process.env.ADMINSTATE } );
 
-	test(
-		'Marketing Overview page have relevant content',
-		{ tag: tags.SKIP_ON_WPCOM },
-		async ( { page } ) => {
-			// Go to the Marketing page.
-			await page.goto(
-				'wp-admin/admin.php?page=wc-admin&path=%2Fmarketing'
-			);
-
-			// Heading should be overview
-			await expect(
-				page.getByRole( 'heading', { name: 'Overview' } )
-			).toBeVisible();
-
-			// Sections present
-			await expect(
-				page.getByText( 'Channels', { exact: true } )
-			).toBeVisible();
+	test( 'Marketing Overview page have relevant content', async ( {
+		baseURL,
+		page,
+		request,
+	} ) => {
+		// See if this is a WPCOM site.
+		const base_hostname = new URL( baseURL ).hostname;
+		const is_wpcom_site = (
+			await request.get(
+				`https://public-api.wordpress.com/wp/v2/sites/${ base_hostname }`
+			)
+		 ).ok();
```

Comment:
Initially, I thought about using these approaches in determining whether it's a WPCOM site or not:
- Using Woo's GET System Status endpoint and checking if `wpcomsh` is listed in the mu plugins from the response.
- Matching `baseURL` with domain names specific to WPCOM.

But I think using the WPCOM public api is simpler and more reliable in this case. All public WPCOM sites can be queried using that public endpoint anyway, and the routes pointing to more sensitive resources like `/users` were protected by that public endpoint when I checked. So the risks from an exposed WPCOM test site URL in our Playwright trace files is low.

Reply:
As mentioned [here](https://github.com/woocommerce/woocommerce/pull/54748#discussion_r1930170572), we'll try to use the approach based on the hostname instead.